packages/fxa-payments-server/server/lib/server.js

```diff
@@ -78,8 +78,9 @@ module.exports = () => {
 
   function listen () {
     const port = config.get('listen.port');
+    const host = config.get('listen.host');
```

Comment:
[Looks like this defaults to `127.0.0.1`](https://github.com/mozilla/fxa/blob/916a0ce72d42d47a3d820d233da307055cfaad24/packages/fxa-payments-server/server/config/index.js#L32), so that's  fine for dev.